### PR TITLE
fix: delete dependency of jackc/pgconn

### PR DESCRIPTION
**Pull request by closetool**
Signed-off-by: closetool <c299999999@qq.com>

Fix: #115

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -5,7 +5,6 @@ go 1.14
 require (
 	github.com/casbin/casbin/v2 v2.28.3
 	github.com/go-sql-driver/mysql v1.5.0
-	github.com/jackc/pgconn v1.8.0
```
Comment (hsluoyz) [inline go.mod:8]: go.sum update?

**Review by hsluoyz — CHANGES_REQUESTED** [review]

File: adapter_test.go [hunk 2]
```diff
@@ -332,7 +333,7 @@ func TestAdapterWithCustomTable(t *testing.T) {
 
 	if err = db.Exec("CREATE DATABASE casbin_custom_table").Error; err != nil {
 		// 42P04 is	duplicate_database
-		if err.(*pgconn.PgError).Code != "42P04" {
+		if !strings.Contains(fmt.Sprintf("%s", err), "42P04") {
```
Comment (hsluoyz) [inline adapter_test.go:336]: Is this strings.contains way safe?

**Review by hsluoyz — CHANGES_REQUESTED** [review]

File: go.mod (hunk 1 above)
Comment (closetool) [reply go.mod:8]: `go mod tidy` used

File: adapter_test.go (hunk 2 above)
Comment (closetool) [reply adapter_test.go:336]: It should be safe.

**Review by hsluoyz — APPROVED** [review]

**Pull request merged**